src/_scss/pages/about/aboutPage.scss

```diff
@@ -0,0 +1,26 @@
+@import "../../mixins/fullSectionWrap";
```

Comment:
Import statements need to happen inside the CSS class block, otherwise the styles being imported are unconstrained - that is automatically applied throughout the site. This is causing other pages to display incorrectly.